### PR TITLE
Make classificatorCode case insensitive 

### PR DESCRIPTION
**Pull request by Mergoth**
Make classificatorCode case insensitive casting it to lowercase.

### CODE REVIEW

**Review by ohhaithere — DISMISSED** [review]
Looks fine by me

**Review by Vladlis — DISMISSED** [review]
Please write a nicer commit message.

File: classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java [hunk 1]
```diff
@@ -85,30 +84,43 @@ public void getClassificatorTypes() throws Exception {
...
 
     @Test
     public void getTopItems() throws Exception {
-        this.mockMvc.perform(get("/OKPD").accept(MediaType.APPLICATION_JSON))
+        this.mockMvc.perform(get("/okpd").accept(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk())
                 .andDo(document("classificator-top-items", preprocessResponse(prettyPrint())));
     }
 
+    /*
```
Comment (Vladlis) [inline classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java]: Is it an example of `prettyPrint`? Please add explanation.

File: classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java [hunk 2]
```diff
@@ -34,9 +34,9 @@
     @Autowired
     private ClassificatorService classificatorService;
     @Autowired
-    private ClassificatorItemService itemService;
+    private SearchService searchService;
```
Comment (Vladlis) [inline classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java:36]: How is this change related?

**Review by Vladlis — DISMISSED** [review]

File: classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java (hunk 2 above)
Comment (Mergoth) [reply classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java:36]: Unrelated, but it's ok to have small simple clean-ups unrelated to PR's, because there's a lot to clean)

File: classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java (hunk 1 above)
Comment (Mergoth) [reply classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java]: nope, removed this garbage

Comment (Vladlis) [reply classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java]: Strange comment is still here

**Review by Vladlis — DISMISSED** [review]
Everything else is ok

**Review by Vladlis — APPROVED** [review]
